K2Bridge/Factories/AggregateFactory.cs

```diff
@@ -138,6 +143,53 @@ public static void AddAggregates(this AggregateDictionary aggregateDictionary, s
             }
         }
 
+        /// <summary>
+        /// Get extended stats for standard deviation aggregate from a given <see cref="DataRow"/>.
+        /// </summary>
+        /// <param name="columnName">The column name.</param>
+        /// <param name="row">The row to be parsed.</param>
+        /// <param name="logger">ILogger object for logging.</param>
+        /// <returns><see cref="ExtendedStatsAggregate"></returns>
+        private static ExtendedStatsAggregate GetExtendedStatsAggregate(string columnName, DataRow row, ILogger logger)
+        {
+            logger.LogTrace("Get extended stats for standard deviation aggregate for {}", columnName);
+
+            var extendedStatsAggregate = new ExtendedStatsAggregate();
+
+            var jObject = (JObject)row[columnName];
+
+            if (jObject == null)
+            {
+                return extendedStatsAggregate;
+            }
+
+            if (jObject.ContainsKey(AggregationsConstants.StandardDeviationBoundsUpper))
+            {
+                var upper = jObject[AggregationsConstants.StandardDeviationBoundsUpper].Value<double?>();
+                extendedStatsAggregate.StandardDeviationBounds.Upper = upper ?? (double.IsNaN(upper.Value) ? null : upper);
+            }
+
+            if (jObject.ContainsKey(AggregationsConstants.StandardDeviationBoundsLower))
```

Comment:
Is there ever a legitimate reason for these keys not to exist? if not, then throw

Reply:
Well, we have no control on the networking, on ADX and so on...
So far, theorically, this shoud not happened, but just in case, having this assumption could avoid some potential issues.

Reply:
My point was to throw an error if these keys don't exist.
Silently discarding them is a bad option.

Reply:
Done